ante/evm/07_can_transfer.go

```diff
@@ -44,16 +45,16 @@ func CanTransfer(
 	}
 
 	stateDB := statedb.New(ctx, evmKeeper, statedb.NewEmptyTxConfig(common.BytesToHash(ctx.HeaderHash())))
-	evm := evmKeeper.NewEVM(ctx, msg, cfg, evmtypes.NewNoOpTracer(), stateDB)
+	evm := evmKeeper.NewEVM(ctx, msg, cfg, nil, stateDB)
 
 	// check that caller has enough balance to cover asset transfer for **topmost** call
 	// NOTE: here the gas consumed is from the context with the infinite gas meter
-	if msg.Value().Sign() > 0 && !evm.Context.CanTransfer(stateDB, msg.From(), msg.Value()) {
+	if msg.Value.Sign() > 0 && !evm.Context.CanTransfer(stateDB, msg.From, uint256.MustFromBig(msg.Value)) {
```

Comment:
Is it actually safe to panic here? Seems like we want to return an error on failure.